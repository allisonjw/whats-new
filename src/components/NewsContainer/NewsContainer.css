section {
    background-color: hotpink;
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
}