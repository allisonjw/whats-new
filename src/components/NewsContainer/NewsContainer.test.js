import React from 'react';
import { shallow } from 'enzyme';
import NewsContainer from './NewsContainer';

describe('NewsContainer', () => {
  it.skip('should match the snapshot', () => {
    const wrapper = shallow(<NewsContainer
      
    />);

    expect(wrapper).toMatchSnapshot();
  });
});