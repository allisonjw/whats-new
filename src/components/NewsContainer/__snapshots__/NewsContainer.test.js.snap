// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsContainer should match the snapshot 1`] = `
<section>
  <NewsArticle
    description="A North Carolina man has died after getting an infection often associated with shellfish, state health officials say."
    headline="Man dies after getting infection linked to shellfish, North Carolina officials say"
    image="https://www.newsobserver.com/entertainment/restaurants/o6w6n7/picture184859643/alternates/FREE_768/MAIN%20OPTION%202:%20OYSTERS"
    key="1"
    url="https://www.bnd.com/news/nation-world/national/article235549897.html"
  />
  <NewsArticle
    description="U.S. health officials, including Health and Human Services Secretary Alex Azar, are encouraging Americans to get their flu shots early this year."
    headline="Azar, other health officials press people to get flu shots early"
    image="https://thehill.com/sites/default/files/styles/thumb_small_article/public/flu_012918getty.jpg?itok=1dygJH3l"
    key="2"
    url="https://thehill.com/homenews/news/463354-azar-other-public-health-officials-press-people-to-get-flu-shots-early"
  />
</section>
`;
